services/core-web/src/components/common/DocumentTable.js

```diff
@@ -32,76 +31,22 @@ const defaultProps = {
   additionalColumnProps: [],
   documentColumns: null,
   documentParent: null,
+  defaultSortKeys: ["upload_date", "dated"], // keys to sort by when page loads
```

Comment:
Not super visible here, but changed the keys that this table uses (now they're default parameters in the document columns file) to match the ones commonly returned by the API so we're not forced to do the same data transformation or overrides every time.